bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -0,0 +1,101 @@
+      </ul>
+
+      <p>We reserve the right not to pay bounties for security bugs in or caused by additional third party software (e.g. binary plugins, extensions) not bundled by Mozilla in a release.</p>
+
+    <table class="table">
+    <thead>
+      <tr>
+        <th>Novel vulnerability and explit, new form of exploitation or an exceptional vulnerability</th>
+        <th>High quality bug report with clearly exploitable critical vulnerability<sub>1</sub></th>
+        <th>High quality bug report of a critical or high vulnerability<sub>2</sub></th>
+        <th>Minimun for a high or critical vulnerability<sub>3</sub></th>
+        <th>Medium vulnerability</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>$10,000+</td>
+        <td>$7,500</td>
+        <td>$5,000</td>
+        <td>$3,000</td>
+        <td>$500 - $2500</td>
+      </tr>
+    </tbody>
+</table>
+<p><sup>1</sup>This is a report that includes details on exploitation of the vulnerability. A good example of this is a vulnerability that can gain remote code execution without the need of another vulnerability. </p>
+
+<p><sup>2</sup>This is a high quality report that includes minimized test cases and clear stack traces.</p>
+
+<p><sup>3</sup> this is a bug that includes a fuzzer report or a crash dump. </p>
+
+      <h2>Claiming a Bug Bounty</h2>
+
+      <p>To claim a bounty:</p>
+      <ul>
+        <li>Make sure you have a Bugzilla account</li>
+          <li>File a bug at https://bugzilla.mozilla.org describing the security issue.</li>
+          <li>When creating the bug, be sure to check the box near the bottom of the entry form that marks this bug report as confidential.</li>
+          <li>Attach a "proof of concept" testcase, or point to explicit code that identifies the vulnerability or exploit.  While not required, such a testcase will help us judge submissions more quickly and accurately.</li>
+          <li>If you have debug output or output from a tool demonstrating the issue, please include it in the bug. (If it is very long, please attach it to the bug as an attached file.)</li>
+          <li>Notify the Mozilla Security Group by email to security@mozilla.org and include the number of the bug you filed and a brief summary of the issue.</li>
+    </ul>
+
+    <p>We ask that you be available to follow along and provide further information on the bug as needed, and invite you to work together with Mozilla engineers in reproducing, diagnosing, and fixing the bug. As part of this process we will provide you full access to participate in our internal discussions about the bug; for more information read our policy for handling security bugs.</p>
```

Comment:
Is there a relevant URL we could link to for the "our policy for handling security bugs" text?